packages/fx-core/src/plugins/resource/frontend/plugin.ts

```diff
@@ -281,26 +281,26 @@ export class FrontendPluginImpl {
     const result: ScaffoldArmTemplateResult = {
       Modules: {
         frontendHostingProvision: {
-          Content: await fs.readFile(moduleFilePath, Constants.BicepFileEncoding),
```

Comment:
NIT: Could you help remove this constant since we don't need it any more？
By the way, I don't think `UTF8Encoding` is a good naming since we will never change its value to `unicode` or others. We apparently follow the convention to extract the string but it brings few benefit.

Reply:
The constant `ConstantString.UTF8Encoding` can be used between all resource plugins. It is a great benefit.

Reply:
Removed the `Constants.BicepFileEncoding`. Thx.